src/index.js

```diff
@@ -0,0 +1,187 @@
+const currentLocation = {
+    city: "Seattle",
+    latitude: 47.6038321,
+    longitude: -122.3300624,
+    climate: temperature,
+};
+
+window.onload = () => {
+    displayCurrentLocation();
+    getCurrentForecast();
+    setTheMood();
+    changeTemperatureAndLandscapeStyling();
+};
+
+
+const displayCurrentLocation = () => {
+    const cityName = document.getElementById("city-name");
+    cityName.textContent = "✨ " + currentLocation.city + " ✨";
+}
+
+
+const updateDisplayForGivenCity = () => {
+    const cityNameInput = document.getElementById("city-name-input");
+    cityNameInput.addEventListener("keyup", () => {
+        currentLocation.city = cityNameInput.value;
+        displayCurrentLocation();
+    });
+}
```

Comment:
For ever cleaner code, consider moving this event listener down to `registerEventHandlers` and refactoring this function.